src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/COMException.cs

```diff
@@ -24,19 +24,19 @@ public COMException()
         }
 
         public COMException(string? message)
-            : base(message)
+            : base(message ?? SR.Arg_COMException)
         {
             HResult = HResults.E_FAIL;
         }
 
         public COMException(string? message, Exception? inner)
-            : base(message, inner)
+            : base(message ?? SR.Arg_COMException, inner)
         {
             HResult = HResults.E_FAIL;
         }
 
         public COMException(string? message, int errorCode)
-            : base(message)
+            : base(message ?? SR.Arg_COMException)
```

Comment:
`Arg_COMException` is too specific.
https://github.com/dotnet/runtime/blob/34ea77705ba9a7fe5ecaf752880a310ba8768d5d/src/libraries/System.Private.CoreLib/src/Resources/Strings.resx#L259-L260

Reply:
Are you proposing reverting changes to this class?

Reply:
I think message should be changed (do not mention EFAIL). It is misleading now, since HResult setter is no longer protected and can be changed after creation.